src/style.css

```diff
@@ -1 +1,92 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(/src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(1) {
+  background-image: url(/src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(2) {
+  background-image: url(/src/images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(1) {
+  background-image: url(/src/images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
Redundant CSS property: `background-repeat: no-repeat;` is repeated in every rule where `background-image` is set. Consider setting this property once for `.stars__star` if it applies to all stars.